src/ispypsa/translator/snapshot.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+
+import pandas as pd
+
+
+def create_snapshot_index(
```

Comment:
Should this be renamed to something like `create_complete_snapshot_index` or `create_full_snapshot_index` to indicate this function is creating a full, pre-aggregation set of snapshots (i.e. there might be a similar function to conduct checks post aggregation)?

Reply:
Good suggestion. I've changed the name to create_complete_snapshot_index.